wiki/hacked-accounts.md

```diff
@@ -0,0 +1,38 @@
+# Getting Live Support for Hacked Accounts
+
+This article covers how to get live chat support in case your account gets hacked.
+
+::: tip Note
+We can not help recovering hacked accounts in the Discord server, as we do not have access to account information.
+:::
```

Comment:
```suggestion
::: tip Note
We can not help you recover hacked accounts in the Discord server, as we do not have access to account information.
:::
```
I prefer it this way, if not a typo.

Reply:
While not a typo, it does look better. Agreed.

Reply:
Just looked it up, it appears that the verb "help" doesn't take a gerund (v. + ing) after it directly, but what's done in this suggestion is a way to write it correctly at least 🤷‍♂️